myosuite/envs/myo/myodm/myodm_v0.py

```diff
@@ -1,21 +1,21 @@
-""" =================================================
+"""=================================================
 Copyright (C) 2018 Vikash Kumar
 Author  :: Vikash Kumar (vikashplus@gmail.com), Sudeep Dasari (sdasari@andrew.cmu.edu), Vittorio Caggiano (caggiano@gmail.com)
 Source  :: https://github.com/MyoHub/myosuite
 License :: Under Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0 Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
-================================================= """
+================================================="""
 
 import collections
 import os
 import time
 
+import mujoco
 import numpy as np
 
 from myosuite.envs.myo.base_v0 import BaseV0
 from myosuite.logger.reference_motion import ReferenceMotion
 from myosuite.utils import gym
-from myosuite.utils.quat_math import euler2quat, mat2quat, quat2euler, quatDiff2Vel
-from myosuite.utils import gym
+from myosuite.utils.mjc import body_name2id, geom_name2id, site_name2id
```

Comment:
MuJoCo now allows named indexing where we can get the indexes using model.site(name)